lib/webpack-manifest-plugin/hooks.js

```diff
@@ -128,13 +128,18 @@ const emitHook = function emit(
 const normalModuleLoaderHook = ({ moduleAssets }, loaderContext, module) => {
   const { emitFile } = loaderContext;
 
+  // assetInfo parameter was added in Webpack 4.40.0
```

Comment:
We don't need this comment

Reply:
I actually added as it is also stated in https://github.com/webdeveric/webpack-assets-manifest/blob/master/src/WebpackAssetsManifest.js#L735, and the fact that I couldn't find any real documentation about this parameter anywhere...

It does clarify why the `Object.assign` is used on line 134.